CartMgmt/Cart/cart.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --white-color: #fff;
+  --grey-color: #808080;
+  --peach-color:rgb(241, 203, 219);
```

Comment:
Use consistent value type for colors either of the one.

Reply:
ok will keep in mind.